crates/holochain_keystore/Cargo.toml

```diff
@@ -43,3 +37,15 @@ holochain_sqlite = { version = "^0.2.0-beta-rc.4", path = "../holochain_sqlite"
 assert_cmd = "2.0.4"
 serde_yaml = "0.9.10"
 tempdir = "0.3.7"
+
+[features]
+default = [ "sqlite" ]
```

Comment:
I wonder if it would be advantageous to leave "sqlite" out of the default features for all crates, so that the flavor has to be explicitly set? The intention being that it's better to have an error than to implicitly make the wrong choice by convention. Though I'm not sure if that would actually be achieved by removing the defaults. It is all a bit mind boggling :)

Reply:
I think it's okay to set default features when they are genuinely a helpful recommendation to the consumer. In this case, `lair_keystore` always requries sqlite and will default to vendoring so by turning off default features for it and asking for bundling instead we avoid consumers of teh `holochain_keystore` crate needing to know that detail. You could equally argue that this wouldn't be necessary if `lair_keystore` didn't set a default that is not desirable in this context though I think.

Similarly for `holochain_sqlite`, it doesn't make sense to import that without `sqlite` so I've left it enabled by default in there.

Not that as per @neonphog's comment above, the `sqlite` and `sqlite-encrypted` can and should be enabled together so this default does not need to be filtered out or otherwise disabled to do a correct production build.